conf/groups.conf

```diff
@@ -235,7 +235,6 @@ groups: (
 		jail: true
 		jailfor: true
 		mute: true
-		storagelist: true
```

Comment:
I mean, this PR is just #2724 rebased, so it has the same caveats as the original. Which includes no default storage and not having implemented some conveniences like `@storagelist` command. Do keep in mind this got the point of "no changes requested by team" before being entirely forgotten by said team and rendered un-mergeable. Most issues you pointed out either weren't considered an issue in the original or are just maintenance upkeep. _Large, complex, non-urgent_ PRs like this one suffer from an unreasonably large maintenance upkeep. Usually something with only two or one or none of these three attributes get in, breaking the PR and thus the high maintenance upkeep. Honorable mention to #466 which is another good example of death by upkeep.

...Although for these design decisions you can ask @dastgirp as they are the _original_ author, I guess? I don't think they'll remember much though, it _has_ been four years already.

(To be more clear and if memory serves as it's been a long time since I looked at the original, `@storagelist` will not work with this PR short of a total rewrite of the command, so it was decided to do a regression and remove the command, as it's seldom used and out of scope, from the discussions if it should list all storages or take an argument, up to the actual implementation which would add even more stuff to review in an already large PR. There was no objection from the original reviewers).

Reply:
I mean, this PR is just #2724 rebased, so it has the same caveats as the original. Which includes no default storage and not having implemented some conveniences like `@storagelist` command. Do keep in mind this got the point of "no changes requested by team" before being entirely forgotten by said team and rendered un-mergeable. Most issues you pointed out either weren't considered an issue in the original or are just maintenance upkeep. _Large, complex, non-urgent_ PRs like this one suffer from an unreasonably large maintenance upkeep. Usually something with only two or one or none of these three attributes get in, breaking the PR and thus the high maintenance upkeep. Honorable mention to #466 which is another good example of death by upkeep.

...Although for these design decisions you can ask @dastgirp as they are the _original_ author, I guess? I don't think they'll remember much though, it _has_ been four years already.

(To be more clear and if memory serves as it's been a long time since I looked at the original, `@storagelist` will not work with this PR short of a total rewrite of the command, so it was decided to do a regression and remove the command, as it's seldom used and out of scope, from the discussions if it should list all storages or take an argument, up to the actual implementation which would add even more stuff to review in an already large PR. There was no objection from the original reviewers).